grpc-proto/proto/schema.proto

```diff
@@ -40,6 +38,27 @@ enum ColumnOrderBy {
   DESC = 0x01;
 }
 
+// INDEXES
+
+message CqlIndex {
+  string name = 1;
+  string columnName = 2;
+  bool custom = 3;
+  string customType = 4;
```

Comment:
Is this really necessary? Leaving String value out should be fine and value would be left as `null` (and there is also method for checking if value was set).
I am not very familiar with use "XxxValue" although have seen it in places so I assume there are legit reasons for using it in some cases.

Reply:
`string` produces an empty string when the value is missing, whereas `StringValue` has the notion of "absent".

https://stackoverflow.com/questions/51707877/what-is-the-point-of-google-protobuf-stringvalue

Reply:
I think that only applies to Go tho -- in Java, `String`s won't be "" but plain `null`. But since Go has no concept of `null` (at least not as universal value like Java) some default value must be used, hence "" on that platform.
Even with Go I think there is access for "isXxxSet()" if someone does have to distinguish Not-passed (absent) from "".

But I guess it's a valid point as we are not counting on only Java being used for implementation and might be more convenient for some languages. Even if possible extra wrapping for Java.

Reply:
> in Java, `String`s won't be "" but plain `null`

Are you sure? I haven't tried decoding a message, but `newBuilder.build()` gives me an empty string, so I assumed that was the default.

Reply:
I was thinking more the case of receiving side where I think undefined would be received as `null`, and for Proto v2 definitely was. But I must admit I do not know how builder does that so you may be right. Proto v3 seems to have changed a lot.

Reply:
fixed